commands/pigeon.rb

```diff
@@ -0,0 +1,51 @@
+class Pigeon < BaseCommand
+  require 'yaml'
+
+  def menu
+    @menu ||= YAML::load_file(File.join(__dir__, 'pigeon.yml')).symbolize_keys
+  end
+
+  def main
+    @main ||= menu[:mains].sample
+  end
+
+  def sample_sides
+    @sample_sides ||= menu[:sides].sample(rand(1..4))
+  end
+
+  def sides(number_of_sides = 4)
```

Comment:
DT,

Do I need to do anything with these?

On Mon, Jun 1, 2015 at 8:36 AM, David Thyresson <notifications@github.com>
wrote:

> In commands/pigeon.rb
> <https://github.com/IntrepidPursuits/KevinBot/pull/5#discussion_r31420999>
> :
>
> > +class Pigeon < BaseCommand
> > +  require 'yaml'
> > +
> > +  def menu
> > +    @menu ||= YAML::load_file(File.join(__dir__, 'pigeon.yml')).symbolize_keys
> > +  end
> > +
> > +  def main
> > +    @main ||= menu[:mains].sample
> > +  end
> > +
> > +  def sample_sides
> > +    @sample_sides ||= menu[:sides].sample(rand(1..4))
> > +  end
> > +
> > +  def sides(number_of_sides = 4)
>
> Ah, yes, it is meant to be a param to
>
> @sample_sides ||= menu[:sides].sample(rand(1..4)) <- the 4
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IntrepidPursuits/KevinBot/pull/5/files#r31420999>.
>